test/test_helper.rb

```diff
@@ -0,0 +1,11 @@
+require "simplecov"
```

Comment:
Glad this works correctly now. So grateful for all our cohort help in getting this working